cmdstanpy/stanfit.py

```diff
@@ -1135,7 +1135,7 @@ def stan_variable(self, name: str) -> np.ndarray:
         if len(col_idxs) > 0:
             shape = self._metadata.stan_vars_dims[name]
         return np.ndarray(
```

Comment:
There's no need to wrap a call to np.array in np.ndarray, so the full return statement for both of these should just be `return np.array(xs).reshape(shape)`